scapy/config.py

```diff
@@ -634,7 +634,7 @@ class Conf(ConfClass):
                    'inet6', 'ipsec', 'ir', 'isakmp', 'l2', 'l2tp',
                    'llmnr', 'lltd', 'mgcp', 'mobileip', 'netbios',
                    'netflow', 'ntp', 'ppi', 'ppp', 'pptp', 'radius', 'rip',
-                   'rtp', 'sctp', 'sixlowpan', 'skinny', 'smb', 'snmp',
+                   'roce', 'rtp', 'sctp', 'sixlowpan', 'skinny', 'smb', 'snmp',
```

Comment:
Your layer should do to `contrib/`. The goal is to keep to minimize the number of layer loaded by default.

Reply:
Done.